idalib/src/strings.rs

```diff
@@ -0,0 +1,54 @@
+use std::marker::PhantomData;
+
+use crate::ffi::strings::{
+    build_strlist, clear_strlist, get_strlist_qty, idalib_get_strlist_item_addr,
+    idalib_get_strlist_item_length,
+};
+use crate::ffi::BADADDR;
+
+use crate::idb::IDB;
+use crate::Address;
+
+pub struct StringList<'a> {
+    _marker: PhantomData<&'a IDB>,
+}
+
```

Comment:
I'd add a type alias for `StringIndex` as we do for bookmarks to keep consistency.

Reply:
Done!